examples/pds_discs_examples/pass_discs_via_pdsobjects.m

```diff
@@ -0,0 +1,95 @@
+% Optimal control example from 
```

Comment:
For fun I translated the pss_discs example now with the cool objects class.
It seems to work without big problems, however the ordering of the states in model.x does not follow the order how I define the ball but the order how the contacts are define, i.e. i get [x1,x3,x2] instead of [x1,x2,x3] (which i find more intutive)
This made later the cost weights and plots inconsistent, but the right order of x will fix all of this